packages/wrangler/src/create-worker-preview.ts

```diff
@@ -147,16 +157,15 @@ export async function createPreviewSession(
 		token: string;
 		prewarm: string;
 	}>(bodyText);
-
-	const { host } = new URL(inspector_websocket);
-	const query = `cf_workers_preview_token=${token}`;
+	const inspector = new URL(inspector_websocket);
+	inspector.searchParams.append("cf_workers_preview_token", token);
 
 	return {
 		id: randomId(),
 		value: token,
-		host,
-		inspectorUrl: new URL(`${inspector_websocket}?${query}`),
-		prewarmUrl: new URL(prewarm),
+		host: ctx.host ?? inspector.host,
```

Comment:
Can the host be `""` (I don't remember if the config validator prevents that) if so this could have unintended behavior.